std/process.d

```diff
@@ -3251,16 +3293,20 @@ private:
     // New variable
     environment["std_process"] = "foo";
     assert (environment["std_process"] == "foo");
+    assert ("std_process" in environment);
```

Comment:
BTW (and I realize you're sticking to existing style) I think Phobos style is to treat `assert` like a function, not a keyword (i.e. no space before `(`).